### PR TITLE
6.6.0

### LINKED ISSUE #112: Need update of autoprefixer dependency to 9.4.5
Hi, would you mind adding autoprefixer 9.4.5 to the dependencies as it adresses a webpack warning that I can't get rid of ? 
post-preset-env is the only dependency requiring autoprefixer 9.4.2 that my project relies on.
https://github.com/postcss/autoprefixer/issues/1179

Issue fixed in https://github.com/postcss/autoprefixer/releases/tag/9.4.5

Thanks for considering my request ! Keep up the good work 🙏

### LINKED ISSUE #117: preset-env much slower than cssnext because of browserslist
I am comparing a build of a small Vue.js application with _postcss-cssnext_ vs. _postcss-preset-env_. On my machine the build takes about 20 s with _cssnext_ and 34 s with _preset-env_.

Here's a direct comparison (same zoom factor):

cssnext

preset-env-zoom

The difference in time required :yum: for the `require()` is probably because _preset-env_ has many many more recursive requirements, I'm going to ignore that for now and concentrate on the difference in build time. So, where is it spending all of its time...

Here's a zoomed in view of the area marked in black in the image above:

2019-02-27-18-45-36-chrome

It seems to spend a lot of time in a function called `browserslist`. I'm wondering if maybe it's possible to cache the results of that somehow?

Here's the relevant part of my webpack config:

```javascript
{
	test: /\.css$/,
	use: [
		'vue-style-loader',
		{ loader: 'css-loader', options: { importLoaders: 2 } },
		{
			loader: 'postcss-loader',
			options: {
				ident: 'postcss',
				plugins: () => [
					require('postcss-preset-env')({ stage: 0 })
				]
			}
		}
	]
}
```

### PR DESCRIPTION
**Pull request by jonathantneal**
- Moved browserslist caching to before each processing.
- Updated `browserslist` to 4.4.2 (minor)
- Updated `autoprefixer` to 9.4.9 (patch)
- Updated `caniuse-lite` to 1.0.30000939 (patch)
- Updated `postcss` to 7.0.14 (patch)
- Updated `postcss-attribute-case-insensitive` to 4.0.1 (patch)

Resolves #112
Resolves #117

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # Changes to PostCSS Preset Env
 
+### 6.6.0 (February 28, 2019)
+
+- Moved browserslist caching to before each processing.
```
Comment (kblomster) [inline CHANGELOG.md]: Do you think it'd make sense to explicitly point out the effect of this change (performance improvement; reduce overhead per file) here? The current line wouldn't make a lot of sense to me if I wasn't already familiar with the internals.

Comment (jonathantneal) [reply CHANGELOG.md]: @kblomster, agreed. Let me revisit this comment.

**Comment by kblomster** [comment]
CI seems to be failing on Windows, but I'm unsure if this is expected...?

**Comment by jonathantneal** [comment]
I’m not against bumping majors, but the scenario that was technically supported was never implied by the documentation — if anything the opposite was implied by https://github.com/csstools/postcss-preset-env#browsers.

I’ll work on a better CHANGELOG.md message.

Also, if for some reason there was a segment of developers who relied on this undocumented behavior, I can push a patch bump to unroll it and then re-release it as a major.

**Comment by kblomster** [comment]
Allright, that makes sense. No objections from me then, assuming that the Windows CI failure isn't a blocker.

Again, thanks for your amazing responsiveness! 👍

**Pull request merged**